web_client/src/components/ControlPanel.vue

```diff
@@ -206,52 +206,64 @@ export default {
+                class="d-flex px-5 py-3"
+                style="flex-direction: column; flex-grow: 3"
+              >
+                <div
+                  class="d-flex"
+                  style="column-gap: 10px"
+                >
+                  <div>
                     Project:
-                  </v-col>
-                  <v-col cols="9">
+                  </div>
+                  <div>
                     {{ currentViewData.projectName }}
-                  </v-col>
-                </v-row>
-                <v-row no-gutters>
-                  <v-col cols="3">
+                  </div>
+                </div>
+                <div
+                  class="d-flex"
+                  style="column-gap: 10px"
+                >
+                  <div>
                     Experiment:
-                  </v-col>
-                  <v-col cols="9">
-                    <div v-if="lockOwner">
-                      <div>
-                        {{ currentViewData.experimentName }}
-                        <UserAvatar
-                          v-if="lockOwner"
-                          :target-user="lockOwner"
-                          as-editor
-                        />
-                      </div>
-                    </div>
-                  </v-col>
-                </v-row>
-                <v-row no-gutters>
-                  <v-col cols="3">
+                  </div>
+                  <div>
+                    {{ currentViewData.experimentName }}
+                    <UserAvatar
+                      v-if="lockOwner"
+                      :target-user="lockOwner"
+                      as-editor
+                    />
+                  </div>
+                </div>
+                <div
+                  class="d-flex"
+                  style="column-gap: 10px"
+                >
+                  <div>
                     Subject:
-                  </v-col>
-                  <v-col cols="9">
+                  </div>
+                  <div>
                     <b>{{ currentViewData.scanSubject || 'None' }}</b>
-                  </v-col>
-                </v-row>
-                <v-row no-gutters>
-                  <v-col cols="3">
+                  </div>
+                </div>
+                <div
+                  class="d-flex"
+                  style="column-gap: 10px"
+                >
+                  <div>
                     Session:
-                  </v-col>
-                  <v-col cols="9">
+                  </div>
+                  <div>
                     <b>{{ currentViewData.scanSession || 'None' }}</b>
-                  </v-col>
-                </v-row>
-              </v-container>
+                  </div>
+                </div>
+              </div>
               <div
                 v-if="scanCachedPercentage < 1 && scanCachedPercentage > 0"
-                class="ma-10"
+                class="px-6 py-8"
+                style="text-align: center"
```

Comment:
I think there's a vuetify class for this, maybe `align-center` IIRC

Reply:
Changed in [efcdc53](https://github.com/OpenImaging/miqa/pull/563/commits/efcdc53c52a74e1f09983e7ecc08393b4cd31f67)

Reply:
It looks like be0b4a95cff91f9a596ed7261c40aa1e336f23b1 actually reverted the change from [efcdc53](https://github.com/OpenImaging/miqa/pull/563/commits/efcdc53c52a74e1f09983e7ecc08393b4cd31f67)